requirements.txt

```diff
@@ -182,11 +197,12 @@ tqdm==4.15.0
     # via wagtail-inventory
 unidecode==1.1.1
     # via wagtail
-urllib3==1.25.9
+urllib3[secure]==1.25.9
```

Comment:
```
urllib3[secure]==1.25.9
```
Probably nothing, but will this change the functionality of other dependent packages?